api/src/models/website.model.js

```diff
@@ -20,6 +20,8 @@ const websiteSchema = new mongoose.Schema(
     title: {
       type: String,
       required: false,
+      minlength: 3,
+      maxlength: 30,
```

Comment:
Just curious to know if these numbers are set randomly or derived from somewhere?

Reply:
I set this restriction and it's consistent with the validation in the front-end program. Let me know if the length does not make sense.